test/models/test_glpn.py

```diff
@@ -0,0 +1,43 @@
+# code to proprely load data here: https://pytorch.org/hub/facebookresearch_pytorchvideo_x3d/
+from transformers import AutoImageProcessor, GLPNForDepthEstimation, DPTForDepthEstimation
+import onnx
+import torch
+import requests
+from PIL import Image
+import numpy as np
+from onnx2kerastl import onnx_to_keras
+from keras_data_format_converter import convert_channels_first_to_last
+from packaging import version
+from collections import OrderedDict
+from typing import Mapping
+import urllib
+
+
+def test_glpn():
+    import sys
+    sys.setrecursionlimit(10000)
+
+    model = GLPNForDepthEstimation.from_pretrained("vinvino02/glpn-kitti")
+    model = model.eval()
+
+    url = "http://images.cocodataset.org/val2017/000000039769.jpg"
+    image = Image.open(requests.get(url, stream=True).raw)
+    image_processor = AutoImageProcessor.from_pretrained("vinvino02/gÏlpn-kitti")
+    inputs = image_processor(images=image, return_tensors="pt")
+    x = inputs.data['pixel_values']
+    print(x.shape)
+    torch.onnx.export(model, x, 'glpn.onnx')
+    onnx_model = onnx.load('glpn.onnx')
+
+    keras_model = onnx_to_keras(onnx_model, ['input.1'], name_policy='attach_weights_name')
```

Comment:
Will this line work for you?
```suggestion
    keras_model = torch2keras(model, pixel_values)
```
If not i suggest writing onnx model to bytesbuffer instead of to file to reduce I/Os:
```suggestion
    temp_f = io.BytesIO()
    torch.onnx.export(model, x, temp_f)
    temp_f.seek(0)
    onnx_model = onnx.load(temp_f)
```